packages/contracts/test/asset_proxy_dispatcher/proxies.ts

```diff
@@ -0,0 +1,378 @@
+            tokenTransferProxyInstance.address,
+            provider,
+        );
+        // Deploy ERC20 V1 Proxy
+        const erc20TransferProxyV1Instance = await deployer.deployAsync(ContractName.ERC20V1Proxy, [
+            tokenTransferProxy.address,
+        ]);
+        erc20TransferProxyV1 = new ERC20Proxy_v1Contract(
+            erc20TransferProxyV1Instance.abi,
+            erc20TransferProxyV1Instance.address,
+            provider,
+        );
+        await erc20TransferProxyV1.addAuthorizedAddress.sendTransactionAsync(assetProxyDispatcherAddress, {
+            from: accounts[0],
+        });
+        await tokenTransferProxy.addAuthorizedAddress.sendTransactionAsync(erc20TransferProxyV1.address, {
+            from: accounts[0],
+        });
+        // Deploy ERC20 Proxy
+        const erc20TransferProxyInstance = await deployer.deployAsync(ContractName.ERC20Proxy);
+        erc20TransferProxy = new ERC20ProxyContract(
+            erc20TransferProxyInstance.abi,
+            erc20TransferProxyInstance.address,
+            provider,
+        );
+        await erc20TransferProxy.addAuthorizedAddress.sendTransactionAsync(assetProxyDispatcherAddress, {
+            from: accounts[0],
+        });
+        // Deploy ERC721 Proxy
+        const erc721TransferProxyInstance = await deployer.deployAsync(ContractName.ERC721Proxy);
+        erc721TransferProxy = new ERC721ProxyContract(
+            erc721TransferProxyInstance.abi,
+            erc721TransferProxyInstance.address,
+            provider,
+        );
+        await erc721TransferProxy.addAuthorizedAddress.sendTransactionAsync(assetProxyDispatcherAddress, {
+            from: accounts[0],
+        });
+        // Deploy zrx and set initial balances
+        const zrxInstance = await deployer.deployAsync(ContractName.DummyToken, constants.DUMMY_TOKEN_ARGS);
+        zrx = new DummyTokenContract(zrxInstance.abi, zrxInstance.address, provider);
+        await zrx.setBalance.sendTransactionAsync(makerAddress, INITIAL_BALANCE, { from: tokenOwner });
+        await zrx.setBalance.sendTransactionAsync(takerAddress, INITIAL_BALANCE, { from: tokenOwner });
+        dmyBalances = new Balances([zrx], [makerAddress, takerAddress]);
+        await zrx.approve.sendTransactionAsync(tokenTransferProxy.address, INITIAL_BALANCE, {
+            from: takerAddress,
+        });
+        await zrx.approve.sendTransactionAsync(tokenTransferProxy.address, INITIAL_BALANCE, {
+            from: makerAddress,
+        });
+        await zrx.approve.sendTransactionAsync(erc20TransferProxy.address, INITIAL_BALANCE, {
+            from: takerAddress,
+        });
+        await zrx.approve.sendTransactionAsync(erc20TransferProxy.address, INITIAL_BALANCE, {
+            from: makerAddress,
+        });
+        // Deploy ck and set initial balances
+        const ckInstance = await deployer.deployAsync(ContractName.DummyERC721Token, constants.DUMMY_ERC721TOKEN_ARGS);
```

Comment:
I know we use token symbols as variable names in our other tests, but we should probably us something more generic like `erc721Instance`. Took me a while to figure out what `ck` stood for :)